### PR TITLE
Update: `CozyCloud.CozyDrive` version 3.28.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20754)

### CODE REVIEW

File: manifests/c/CozyCloud/CozyDrive/3.28.0/CozyCloud.CozyDrive.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿# Created using YamlCreate.ps1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+
+PackageIdentifier: CozyCloud.CozyDrive
+PackageVersion: 3.28.0
+PackageLocale: en-US
+Publisher: CozyCloud
```
Comment (ItzLevvie) [inline manifests/c/CozyCloud/CozyDrive/3.28.0/CozyCloud.CozyDrive.locale.en-US.yaml:7]:
```suggestion
Publisher: Cozy Cloud
```

File: manifests/c/CozyCloud/CozyDrive/3.28.0/CozyCloud.CozyDrive.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+PublisherUrl: https://cozy.io/en/
+PublisherSupportUrl: https://cozy.io/en/support/
+PrivacyUrl: https://cozy.io/en/privacy/
+#Author: 
+PackageName: CozyDrive
```
Comment (ItzLevvie) [inline manifests/c/CozyCloud/CozyDrive/3.28.0/CozyCloud.CozyDrive.locale.en-US.yaml:12]:
```suggestion
PackageName: Cozy Drive
```

File: manifests/c/CozyCloud/CozyDrive/3.28.0/CozyCloud.CozyDrive.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: machine
```
Comment (ItzLevvie) [inline manifests/c/CozyCloud/CozyDrive/3.28.0/CozyCloud.CozyDrive.installer.yaml:15]:
```suggestion
  Scope: user
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by vedantmgoyal2009** [comment]
Can you please tell how to decide whether to keep `Scope: machine`or `Scope: user`

**Review by ItzLevvie — APPROVED** [review]

**Comment by denelon** [comment]
@vedantmgoyal2009 the scope has to do with whether the package installs for "all users" - "machine" or if it installs for only "one user" - "user". Some installers support both, while some only support one. It's a case by case basis.

**Comment by vedantmgoyal2009** [comment]
Thanks. In some package, the software is installed for only the current user but it requires `machine` otherwise Azure Pipelines would fail.

**Comment by denelon** [comment]
@vedantmgoyal2009 please don't use the scope for the scenario where "machine" would be indicated due to UAC requirements. We would prefer to have other mechanisms to indicate UAC required. https://github.com/microsoft/winget-cli/issues/1284

This would be a bad experience for a user expecting the package to install for all users.

**Pull request merged**